raiden/transfer/mediated_transfer/events.py

```diff
@@ -472,13 +441,11 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'recipient': to_checksum_address(self.recipient),
             'channel_identifier': self.queue_identifier.channel_identifier,
             'message_identifier': self.message_identifier,
-            'payment_identifier': self.payment_identifier,
-            'token_address': to_checksum_address(self.token),
-            'balance_proof': self.balance_proof,
-            'balance_hash': serialization.serialize_bytes(self.balance_proof.balance_hash),
-            'lock': self.lock,
-            'initiator': to_checksum_address(self.initiator),
-            'target': to_checksum_address(self.target),
+            'transfer': self.transfer,
+            'balance_proof': self.transfer.balance_proof,
```

Comment:
Why do you serialize  `balance_proof` and `balance_hash` here? Is this for querying from SQlite?

Reply:
Yes. However, I didn't review the current SQL to check if was necessary or not. @rakanalh  should this be removed?

Reply:
It should be removed, because [here](https://github.com/raiden-network/raiden/blob/master/raiden/transfer/utils.py#L27) we are querying for balance_hash value and `balance_proof.X` values as well.... i'll check out later on what i can do to improve this as Augusto and I know that we had some inconsistencies in the structure of state changes / events ... etc that we had to do this.